### PR TITLE
bugfix/426-Privacy-modal-responsive

### PR DESCRIPTION
**Pull request by waltersajtos**
## Description

Quick fix for the privacy modal, I also went ahead and updated the delete-generic-modal because it had the same issue.

## Type of change

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Open an incognito tab
2. Go to the login page
3. Click FHICT and check if the privacy modal looks good on desktop and mobile

## Link to issue

Closes: #426

### CODE REVIEW

**Review by RubenFricke — COMMENTED** [review]
Implementation and result look great, thanks! 
Although, a small request (maybe a little bit out of scope), the alerts aren't responsive as well. It's up to you whether you want to pick this up in this issue or create a new issue. 
image
Good work 🔥

**Comment by waltersajtos** [comment]
image
image
@RubenFricke

File: src/app/components/alert/alert.component.scss [hunk 1]
```diff
@@ -16,8 +16,17 @@
...
-    box-shadow: 0 3px 6px #0000001a;
-    pointer-events: all;
+  max-width: 600px;
+  width: 100%;
+  margin: 0 auto 10px auto;
+  box-shadow: $box-shadow;
+  pointer-events: all;
+
+  @media only screen and (max-width: 600px) {
+    max-width: 60%;
```
Comment (RubenFricke) [inline src/app/components/alert/alert.component.scss:29]: Why set the max-width to 60%? 
image
And adding margin would improve the experience as well in my opinion.

**Review by RubenFricke — CHANGES_REQUESTED** [review]
The implementation and code look good as usual. One note I had to make.

**Review by RubenFricke — APPROVED** [review]
Everything works, thanks. I updated your CHANGELOG because the issue has grown a little and you accidentally added this change to the wrong release in the CHANGELOG file 😉

**Review by niraymak — APPROVED** [review]
Great work! 👍

**Review by RubenFricke — APPROVED** [review]

**Review by niraymak — APPROVED** [review]

**Pull request merged**